### PR TITLE
Update after first teaching

### LINKED ISSUE #25: Add multiple choice questions
I found that these fit online workshops very well and currently there are none

### PR DESCRIPTION
**Pull request by BeastyBlacksmith**
This lesson was tought on 10.08.2021 which resulted in these improvements.

Fix #25

### CODE REVIEW

**Comment by BeastyBlacksmith** [comment]
@ocaisa how do I tell the spellchecker that this is a false positive?

**Comment by ocaisa** [comment]
> @ocaisa how do I tell the spellchecker that this is a false positive?

Change https://github.com/carpentries-incubator/julia-novice/blob/gh-pages/.github/workflows/lesson.yml#L84 to the list `rouge,ans` (I think)

**Comment by BeastyBlacksmith** [comment]
That worked! Thanks

**Comment by BeastyBlacksmith** [comment]
@ElOceanografo I quoted your [discourse post](https://discourse.julialang.org/t/what-is-the-advantage-of-julia-over-fortran/65964/101) since I really think it nails what I couldn't explain :D 
Please check if you are fine with it or if I should change anything.

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -33,4 +34,23 @@ But before each function is executed it will compile it _just in time_.
 
 Thus you get the flexibility of an interpreted language and the execution speed of the compiled language at the cost of waiting a bit longer for the first execution of any function.
 
+There is another aspect of julia that makes it interesting and that is the way packages compose.
+This is captured the best by an analogy from [Sam Urmy](https://github.com/ElOceanografo):
```
Comment (BeastyBlacksmith) [inline _episodes/01-introduction.md:38]: it starts here

Comment (ElOceanografo) [reply _episodes/01-introduction.md:38]: No complaints, thanks for checking!

**Pull request merged**